content/en/news/releases/1.14.x/announcing-1.14/change-notes/index.md

```diff
@@ -0,0 +1,176 @@
+
+In this fix, we introduce a new struct `consolidatedDestRule` and record all the `destinationrules`' meta
+to avoid missing any `destinationRule` dependencies.  ([Issue #38082](https://github.com/istio/istio/issues/38082))
+
+- **Fixed** an issue causing traffic from a gateway to a service with an [undeclared protocol](/docs/ops/configuration/traffic-management/protocol-selection/#automatic-protocol-selection) being treated as TCP traffic rather than HTTP.
+  ([Issue #37196](https://github.com/istio/istio/issues/37196))
+
+- **Fixed** an issue with `DNS` type `ServiceEntry`s causing excessive DNS requests when the DNS lookup fails.
+  ([Issue #35603](https://github.com/istio/istio/issues/35603))
+
+- **Fixed** IP family detection when using the CNI to behave the same way as without it.
+  ([Issue #36871](https://github.com/istio/istio/issues/36871))
+
+- **Fixed** IPv6 detection on clusters with IPv4 NAT implementation, such as Amazon EKS, by excluding link-local addresses from detection.
+  ([Issue #36961](https://github.com/istio/istio/issues/36961))
+
+## Security
+
+- **Added** a new approach for CA integration through the Envoy SDS API.
+  ([usage]( https://istio.io/latest/docs/ops/integrations/spire/))([design]( https://docs.google.com/document/d/1zJP6QJukLzckTbdY42ZMLkulGXz4gWzH9SwOh4xoe0A)) ([Issue #37183](https://github.com/istio/istio/issues/37183))
+
+- **Added** support for using `PrivateKeyProvider` in SDS. See [#35809](https://github.com/istio/istio/issues/35809)
+
+- **Added** support for TLS configuration API for workloads.  ([Issue #2285](https://github.com/istio/api/issues/2285))
+
+- **Fixed** the request authentication policy to correctly always allow the CORS preflight request.
+  ([Issue #36911](https://github.com/istio/istio/issues/36911))
+
+## Telemetry
+
+- **Added** the implementation of the OpenTelemetry access log.
+
+- **Added** environment variable support at Wasm extension via VM configuration in WasmPlugin API.
+
+- **Added** `WorkloadMode` selection to Logging.
+
+## Extensibility
+
+- **Added** support for WasmPlugin pulling image from private repository with `imagePullSecret`.
+
+## Installation
+
+- **Added** support of installing gateway helm chart as `daemonset`.
+  ([Issue #37610](https://github.com/istio/istio/issues/37610))
+
+- **Fixed** an issue of Envoy losing connection after `istio-ca-root-cert` is changed.
+  ([Issue #36723](https://github.com/istio/istio/issues/36723))
+
+- **Fixed** an issue that was preventing the operator from updating deployments when `.autoscaleEnabled` is `true` and `.k8s.replicaCount` is nonzero.
+  When both `autoscale` is enabled and `replicaCount` is nonzero, warning messages will be generated during validation.
+
+- **Fixed** an unknown field `customService` in `v1alpha1.EgressGatewayConfig`.
+  ([Issue #37260](https://github.com/istio/istio/issues/37260))
+
+- **Fixed** removing `caBundle` default value from Chart to allow a GitOps approach.
```

Comment:
Not sure if we have a Removed section, but if we do, this could be **Removed** `cabundle`

Reply:
Done, this item is moved to the **Removed** section: **Removed** `caBundle ...